performance/models.py

```diff
@@ -35,11 +40,6 @@ class Frequency:
 
 
 class Indicator(models.Model):
-    """The indicator values available for a indicator in a department in a financial year"""
```

Comment:
you don't like my docs? :)

Reply:
I don't remember why I removed it but probably I removed an extra line while removing `unique_together`